financial-services-accelerator/accelerators/fs-is/carbon-home/repository/resources/conf/templates/repository/conf/financial-services.xml.j2

```diff
@@ -18,6 +18,18 @@
  -->
 
 <Server xmlns="http://wso2.org/projects/carbon/financial-services.xml"  xmlns:svns="http://org.wso2.securevault/configuration">
+    <Admin>
+        {% if financial_services.admin.username is defined %}
+        <Username>{{financial_services.admin.username}}</Username>
+        {% else %}
+        <Username>is_admin@wso2.com</Username>
```

Comment:
can't we copy the toml config alias used for iam admin credentials for financial-services.xml ,if someone not defined specifically for financial-services.xml

Reply:
in the toml we have configured to get the iam admin credentials